### ISSUE TITLE
Prime (') symbol in atom name

### ISSUE THREAD

**Issue by gbayarri** [arose/ngl#370]
I'm trying to represent this PDB file:

HETATM 4635  **C3'** BPG A   1      49.533  82.766  53.536  1.00 17.33           C  
HETATM 4636  **O3'** BPG A   1      50.688  81.981  53.415  1.00 21.50           O  
HETATM 4637  **C2'** BPG A   1      50.128  84.079  53.994  1.00 15.61           C  
HETATM 4638  C12 BPG A   1      49.009  84.977  54.509  1.00 16.58           C  
HETATM 4639  **C1'** BPG A   1      48.333  84.409  55.780  1.00 16.82           C  
HETATM 4640  N9  BPG A   1      47.200  85.204  56.278  1.00 16.03           N  
HETATM 4641  C4  BPG A   1      45.903  85.363  55.703  1.00 14.66           C  
HETATM 4642  N3  BPG A   1      45.462  84.786  54.595  1.00 12.56           N  
HETATM 4643  C2  BPG A   1      44.169  85.040  54.341  1.00 13.59           C  
HETATM 4644  N2  BPG A   1      43.617  84.472  53.228  1.00 16.32           N  
HETATM 4645  N1  BPG A   1      43.370  85.890  55.069  1.00 14.33           N  
HETATM 4646  C6  BPG A   1      43.808  86.524  56.224  1.00 14.72           C  
HETATM 4647  O6  BPG A   1      43.050  87.250  56.844  1.00 12.55           O  
HETATM 4648  C5  BPG A   1      45.192  86.201  56.521  1.00 14.85           C  
HETATM 4649  N7  BPG A   1      46.001  86.604  57.584  1.00 17.67           N  
HETATM 4650  C8  BPG A   1      47.132  85.979  57.381  1.00 16.81           C  
HETATM 4651  C04 BPG A   1      45.389  81.838  50.488  1.00 15.62           C  
HETATM 4652  C05 BPG A   1      45.899  81.921  51.803  1.00 13.08           C  
HETATM 4653  C06 BPG A   1      45.324  82.795  52.708  1.00 13.65           C  
HETATM 4654  C01 BPG A   1      44.236  83.572  52.332  1.00 15.32           C  
HETATM 4655  C02 BPG A   1      43.718  83.481  51.018  1.00 15.30           C  
HETATM 4656  C03 BPG A   1      44.283  82.607  50.108  1.00 13.21           C

Due to the prime symbols in the first, second, third and fifth rows, the representations is the following:

![screenshot](https://user-images.githubusercontent.com/19928100/29517538-ea7d2c90-8675-11e7-89d0-bb90f932f662.png)

If I add backslashes to the prime symbols, then the representation is correct, but I think that's not the best practice.

Is there any parameter in NGL that warns that the file must be parsed considering the prime symbols?

Thanks.

**Comment by arose** [comment]
the group was wrongly detected as a coarse grained DNA residue, fixed it by adding better cg detection (check atom count to avoid false positives, see 71f072f060e9ef4a7a2cfd7159bf220ff02424b4)

**Comment by gbayarri** [comment]
Hi Alexander, it works. 

Just a thing: After changing from 0.10.5-13 to 0.10.5-19 I've detected that there is a small delay charging PDB files, and if I start an animation automatically after loading the file (it doesn't matter if it's a setSpin or a representation as a trajectory), the file is not shown until I pass the mouse over the viewport. I've fixed that putting the call to animation inside a setTimeOut of 1 second. Should I report that as a new bug?

Thanks one more time.
